extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -346,7 +346,7 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
       const recipient = orderedRecipients[processed];
       const escapedTitle = Xss.escape(recipient.element.getAttribute('title') || '');
       const nameOrEmail = recipient.name || recipient.email || recipient.invalid || '';
-      const emailHtml = `<span class="email_address ${recipient.element.className}" title="${escapedTitle}">${Xss.escape(nameOrEmail)}</span>`;
+      const emailHtml = `<span class="email_address ${recipient.element.className}" title="${escapedTitle}">${Xss.stripEmojis(nameOrEmail)}</span>`;
```

Comment:
no need for `stripEmojis` here, let's change it back to `escape`

Reply:
will do - thanks!